README.md

```diff
@@ -45,9 +45,15 @@ Mystikos.
 ![](./arch.png)
 
 
-# Installation Guide for Ubuntu 18.04
+# Installation Guide for Ubuntu
 
-## Install Intel SGX DCAP Driver if necessary
+As of release version 0.7.0, Mystikos may be built and installed on Ubuntu 20.04.
```

Comment:
Should still mention all the versions that are compatible so users know that 18.04 is still supported.